README.md

```diff
@@ -1,59 +1,119 @@
+</p>
 
+# Scapy
 
 [![Travis Build Status](https://travis-ci.org/secdev/scapy.svg?branch=master)](https://travis-ci.org/secdev/scapy)
 [![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/secdev/scapy?svg=true)](https://ci.appveyor.com/project/secdev/scapy)
 [![Codecov Status](https://codecov.io/gh/secdev/scapy/branch/master/graph/badge.svg)](https://codecov.io/gh/secdev/scapy)
 [![Join the chat at https://gitter.im/secdev/scapy](https://badges.gitter.im/secdev/scapy.svg)](https://gitter.im/secdev/scapy?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-Scapy is a powerful Python-based interactive packet manipulation
-program and library.
 
-It is able to forge or decode packets of a wide number of protocols,
-send them on the wire, capture them, match requests and replies, and
-much more.
-
-It can easily handle most classical tasks like scanning, tracerouting,
-probing, unit tests, attacks or network discovery (it can replace
-hping, 85% of nmap, arpspoof, arp-sk, arping, tcpdump, tethereal, p0f,
-etc.). It also performs very well at a lot of other specific tasks
-that most other tools can't handle, like sending invalid frames,
-injecting your own 802.11 frames, combining technics (VLAN hopping+ARP
-cache poisoning, VOIP decoding on WEP encrypted channel, ...),
-etc.
-
-# Installation #
-
-Scapy works without any external Python modules on Linux and BSD like operating systems. On Windows, you need to install some mandatory dependencies as described in the documentation.
-
-To benefit from all Scapy features, you might want to install Python modules, such as matplotlib or cryptography.
-
-See the [tutorial](http://scapy.readthedocs.io/en/latest/installation.html) and follow the instructions to install scapy.
-
-## Tutorial ##
-
-To begin with scapy, see the
-[interactive tutorial](http://scapy.readthedocs.io/en/latest/usage.html#interactive-tutorial)
-and
-[the quick demo: an interactive session](http://scapy.readthedocs.io/en/latest/introduction.html#quick-demo)
-(some examples may be outdated).
-
-# Contributing #
+Scapy is a powerful Python-based interactive packet manipulation program and
+library.
+
+It is able to forge or decode packets of a wide number of protocols, send them
+on the wire, capture them, match requests and replies, and much more. It
+designed to allow fast packet prototyping by using default values that work.
+
+It can easily handle most classical tasks like scanning, tracerouting, probing,
+unit tests, attacks or network discovery (it can replace `hping`, 85% of `nmap̀`,
+`arpspoof`, `arp-sk`, `arping`, `tcpdump`, `wireshark`, `p0f`, etc.). It also
+performs very well at a lot of other specific tasks that most other tools can't
+handle, like sending invalid frames, injecting your own 802.11 frames, combining
+techniques (VLAN hopping+ARP cache poisoning, VoIP decoding on WEP protected
+channel, ...), etc.
+
+## Hands-on
+
+###### Interactive shell
```

Comment:
Not sure if that matters in the markdown rendering, but is there a reason for using level 6 `######` titles rather than level 3 `###`?

Reply:
I could not make it work with `### Hands-on` only =)